crates/uv-python/src/discovery.rs

```diff
@@ -1713,6 +1713,15 @@ impl PythonVariant {
     }
 }
 impl PythonRequest {
+    /// Return the [`PythonVariant`] of the request, if any.
+    pub fn variant(&self) -> Option<PythonVariant> {
+        match self {
+            Self::Version(version) => version.variant(),
+            Self::ImplementationVersion(_, version) => version.variant(),
+            _ => None,
```

Comment:
We generally prefer to list all the variants so if we add a new one we're forced to revisit this and see if it needs to be updated.